First/lesson_4.py

```diff
@@ -0,0 +1,41 @@
+def my_new_decorator(function_to_decorate):
+    def the_wrapper_around_the_original_function(n):
```

Comment:
Like we discuss, better use @functools.wraps(function_to_decorate) - for al decorator.